app/core/simulation.ts

```diff
@@ -1521,12 +1512,9 @@ export default class Simulation extends Schema implements ISimulation {
       )
 
       if (this.winnerId === this.redPlayerId) {
-        if (this.bluePlayerId !== "pve" && !this.isGhostBattle) {
-          this.redPlayer.addMoney(1, true, null)
-          client?.send(Transfer.PLAYER_INCOME, 1)
-        }
-      } else if (!this.isGhostBattle) {
-        // if blue player won against ghost, do not make red player take damage
+        this.redPlayer.addMoney(1, true, null)
```

Comment:
I checked, pve team is always redPlayer, just like ghost player is always redPlayer. So this condition was not necessary

Reply:
👍